phpBB/styles/prosilver/theme/common.css

```diff
@@ -5,8 +5,6 @@
 /* stylelint-disable selector-no-qualifying-type */
 
 html {
-	font-size: 100%;
```

Comment:
IMO removing this doesn't change anything. But setting it to 90% rescales the base font size for the whole forum to _normal_ sizing